index.html

```diff
@@ -2,7 +2,11 @@
 
 <head>
   <!-- jQuery -->
-  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+  <script
+  src="https://code.jquery.com/jquery-3.5.1.min.js"
+  integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0="
```

Comment:
What is this sum?

Reply:
This all about to add JQuery 3.x version instead of current 1.12.4) not such necessary, but may be helpful) anyway, as u wish)

Reply:
I'm good with newer version but why not just
```html
<script src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
```
?